site/src/components/Screens/SendLoginCodeScreen.tsx

```diff
@@ -98,7 +114,7 @@ export const SendLoginCodeScreen: VFC<SendLoginCodeScreenProps> = ({
           inputRef={emailInputRef}
           fullWidth
           label="Email address"
-          placeholder="you@example.com"
+          placeholder="claude@shannon.com"
```

Comment:
This is a real non-exampley website, I don't think we should be using it. maybe `claude@example.com`